src/system/SystemLayer.h

```diff
@@ -49,6 +49,12 @@
 namespace chip {
 namespace System {
 
+struct LambdaBridge
+{
+    void (*LambdaProxy)(const void * context);
+    char * LambdaBody[36];
```

Comment:
There's a slack discussion started by @tcarmelveilleux about high memory usage by CHIP events given that the structure is shared across all event types. Perhaps, it would be possible to make the static lambda storage configurable and if not, allocate the lambda body on the heap. Thoughts?

Reply:
I'd like to agree with it. I use 36 here mostly because that I saw the address struct in the union. I'll make it configurable using a macro.